src/appMain/smartDeviceLink.ini

```diff
@@ -185,6 +185,7 @@ ForceUnprotectedService = Non
 ; The PTU will be triggered in case expiration date of certificate
 ; then certain hours amount
 UpdateBeforeHours = 24
+SecurityLevel = 1
```

Comment:
@VadymLuchko please provide a comment what is it for (like for the rest of params here)

Reply:
fixed in be9c3c1340e374d266f87fb35a5048dfe803a4b2